proxy/core/src/main/scala/io/cloudstate/proxy/eventsourced/EventSourcedEntity.scala

```diff
@@ -306,7 +306,10 @@ final class EventSourcedEntity(configuration: EventSourcedEntity.Configuration,
       throw error
 
     case SaveSnapshotSuccess(metadata) =>
-    // Nothing to do
+      // TODO: is it he right place delete the oldest snapshots and events?
+      // TODO ideally we want a one to one mapping between event and snapshot.
+      deleteSnapshots(SnapshotSelectionCriteria().copy(maxSequenceNr = metadata.sequenceNr - 1))
```

Comment:
Why would you want to do this? (esp. for all EventSourcedEntities?)

Reply:
@viktorklang I did it this way just to show how it should work for the the eventually new CrudEntity. I would like to have an one to one mapping between snapshot and event. in this case snapshot and event are almost the same.